### PR TITLE
Add tool to log events for use in cypress tests

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

File: docker/localstack/localstack-init.sh [hunk 1]
```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-set -e
```
Comment (hawx) [inline docker/localstack/localstack-init.sh:3]: Sometimes localstack will fail to create things if they already exist, which breaks things that get setup after, which is very annoying. So I'm removing this to get the whole script to execute

File: internal/event/events.go [hunk 2]
```diff
@@ -37,6 +37,6 @@ type PreviousApplicationLinked struct {
 type ReducedFeeRequested struct {
 	UID              string   `json:"uid"`
 	RequestType      string   `json:"requestType"`
-	Evidence         []string `json:"evidence"`
+	Evidence         []string `json:"evidence,omitempty"`
```
Comment (hawx) [inline internal/event/events.go:40]: https://github.com/ministryofjustice/opg-event-store/pull/15 when I wrote the Cypress test we were sending `null` here, which didn't match the spec either. So, first bug found by the new tool!

File: cmd/event-logger/main.go [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+)
+
+type message struct {
+	Time       time.Time
+	Detail     json.RawMessage
+	DetailType string `json:"detail-type"`
+	Source     string
+}
+
+const messageCount = 10
```
Comment (acsauk) [inline cmd/event-logger/main.go:31]: I had to parse around the code below a bit to work out the purpose of this - would `messageLimit` or `maxMessages` be any clearer on intent?

File: docker/localstack/localstack-init.sh (hunk 1 above)
Comment (acsauk) [reply docker/localstack/localstack-init.sh:3]: I didn't realise it was this opt that was getting in the way - no more `down` then `up` 🙌

**Review by acsauk — APPROVED** [review]
This is so cool - feels much safer now to be able to build events into our feature tests.

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**